### ISSUE TITLE
jwm-2.4.1 broken fonts and not maximizing?

### ISSUE THREAD

**Issue by PrinceKael** [void-linux/void-packages#35962]
### System

* xuname:  
  *nothing comes up here but uname=5.15.26 _1 x86_64*

* package:  
  *affected package(s) including the version*:  jwm-2.4.1_1

### Expected behavior

Windows and panel fonts should be whatever they were set to instead of something...ugly and barely readable. And the windows don't maximise, there's a few pixel of space on x/y-axis which is quite annoying when using a browser.

**Comment by tibequadorian** [comment]
>Windows and panel fonts should be whatever they were set to instead of something...ugly and barely readable.

"ugly and barely readable" is a matter of opinion but I opened a new PR which should fix that.

>And the windows don't maximise, there's a few pixel of space on x/y-axis which is quite annoying when using a browser.

Can't reproduce windows not maximising.
These window borders look like a feature to me, consult upstream if you don't like it.

**Comment by PrinceKael** [comment]
Thanks mate i just panicked and I'm a bit of a newb.ie. I appreciate the link, I went into system.jwmrc and:

1. Went to all the font groups and changed from some "Sans-9" to my ideal font `` "-adobe-helvetica-bold-r-normal-*-14-*-*-*-*-*-iso10646-*``

2. Then I added a group for for ``<Option>nomaxborder</Option>``

which altogether 'fixed' my issue and brought back my settings some thing a lot more usable. ``jwm -p`` isn't ruturning any errors now apart from some "Segmentation fault" but at least now it looks clean and I've set things up better.

It seems an update just changed things from my usual set-up. I'll change anything else as needed but it's now close enough to what wanted originally, so thanks for the links.
